app/src/pages/inside/common/dynamicFieldsBlock/dynamicFieldsBlock.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import moment from 'moment';
+import DatePicker from 'react-datepicker';
+import { Input } from 'components/inputs/input';
+import { InputTagsSearch } from 'components/inputs/inputTagsSearch';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { FormField } from 'components/fields/formField';
+import { TEXT_TYPE, ARRAY_TYPE, DROPDOWN_TYPE, DATE_TYPE, DATE_FORMAT } from './constants';
+import styles from './dynamicFieldsBlock.scss';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class DynamicFieldsBlock extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    fields: PropTypes.array,
+    customBlock: PropTypes.func,
+    withFormValidation: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    fields: [],
+    customBlock: null,
+    withFormValidation: false,
+  };
+
+  getInputOptions = (values) =>
```

Comment:
You can use default parameter value instead of `(values || [])` and destructuring assignment like:
`getInputOptions = (values = []) => values.map(({ value, label }) => ({ value, label }));`

Reply:
Of course! Thanks. Fixed.